libs/perun/utils/src/lib/table-wrapper/table-wrapper.component.ts

```diff
@@ -46,5 +55,6 @@ export class TableWrapperComponent implements OnInit {
     if (this.tableId) {
       this.tableConfigService.setTablePageSize(this.tableId, event.pageSize);
     }
+    this.topNav.nativeElement.scrollIntoView(false); // scroll to top of table on page changes
```

Comment:
```suggestion
    this.topNav.nativeElement.scrollIntoView(true); // scroll to top of table on page changes
```
I would prefer the `true` parameter because if you have more tables on one page, it looks better with `true` option IMHO (you can test it on /myProfile/roles -> Membership section).

Maybe there is one better option that looks the best for me. 
```suggestion
    // scroll to top of table on page changes
    this.topNav.nativeElement.scrollIntoView({
      behavior: 'smooth',
      block: 'start',
      inline: 'nearest',
    });
```

Reply:
I like this option the most  as well. Initialy I thought scrolling would be annoying with 100 items per page but the behaviour is truly "smooth".